.babelrc

```diff
@@ -0,0 +1,8 @@
+{
+  "presets": ["react-app","es2015"],
+  "plugins": [
+      [ "react-intl", {
+          "messagesDir": "build/messages"
```

Comment:
is this possible to be added in the webpack config?

Reply:
will figure out